packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -305,6 +305,8 @@ class InteractiveGraphEditor extends React.Component<Props> {
                 lockedFigures: this.props.lockedFigures,
                 fullGraphAriaLabel: this.props.fullGraphAriaLabel,
                 fullGraphAriaDescription: this.props.fullGraphAriaDescription,
+                // Set the "correct answer" graph to static when editing is disabled
```

Comment:
Can you also add this comment above `static` in the grapher editor? I think the context is helpful.